common/src/Model/TransformObjectVisitor.h

```diff
@@ -22,22 +22,30 @@
 
 #include "FloatType.h"
 #include "Model/NodeVisitor.h"
+#include "Model/Object.h"
+
+#include <optional>
 
 namespace TrenchBroom {
     namespace Model {
         class TransformObjectVisitor : public NodeVisitor {
```

Comment:
We might want to document that is stops transforming the nodes on the first error, so it's the responsibility of the user to clean up if it fails part way through transforming a list of nodes.

Reply:
Yeah, agree.

Reply:
Added a comment in b604af43b.